features/step_definitions/helper_services.rb

```diff
@@ -714,3 +714,33 @@
   step 'the step should succeed'
   step 'the pod named "cluster-capacity" becomes ready'
 end
+
+# example of machine auto-scaler https://github.com/openshift-qe/output_references/blob/master/autoscale/machine-autoscaler.yaml
+Given /^I enable autoscaling for my cluster$/ do
+  name_place_holder = ""
+  base_template_hash = {
+    "apiVersion" => "autoscaling.openshift.io/v1beta1",
+    "kind" => "MachineAutoscaler",
+    "metadata" => {
+      "name" => name_place_holder,
```

Comment:
we don't need a separate variable for a placeholder. You could skip setting that value and set it afterwards. Or set to hardcoded value `nil` or empty string `""`. Any special need to use a variable?

Reply:
the name has to match the node

Reply:
I'm not arguing against that. Just line 742 will still work if you remove lines 720 and 725. Or am I missing something?

Reply:
@akostadinov ah ok, I was not clear on what you mean...yeah, I can do that...